### DISCUSSION TITLE
BLS appointment book bot

### DISCUSSION THREAD

**Discussion by anjamwaqas** [community/community#147484] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hello every one i am searching for PLS appointment booking bot because i want to book my appointment only one for me unfortunately the agents demanding too much money because they have bot software, i am an industrial automation engineer and i have know of PLC DCS and SCADA programming, please help me to book only one appointment for me

**Comment by shinybrightstar** [comment]
Hi @anjamwaqas 

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
